src/librbd/ImageState.h

```diff
@@ -25,8 +25,8 @@ class ImageState {
   ImageState(ImageCtxT *image_ctx);
   ~ImageState();
 
-  int open();
-  void open(Context *on_finish);
+  int open(bool dont_open_parent=false);
```

Comment:
Nit: can we rename this to something like ```skip_open_parent``` everywhere? Something about "dont" feels a bit off